docs/direct-access-urls.md

```diff
@@ -182,6 +194,17 @@ The pre-signed request generates a download for the remote content.
 >**Known Limitations:** DAU generation on S3 depends on the security credentials used:
 > https://docs.aws.amazon.com/AmazonS3/latest/dev/ShareObjectPreSignedURL.html
 
+
+##### Azure Connector
```

Comment:
This looks useful. Also add similar section for S3 Connector (with links to S3 docs) ?

Reply:
Looks like similar section is right above or I misunderstood you.

Reply:
Yes, you're right - I missed it.